src/components/WebexMemberRoster/WebexMemberRoster.scss

```diff
@@ -26,11 +37,11 @@
     align-items: center;
     padding-right: 0.375rem;
 
-    .title {
+    .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-title {
```

Comment:
we target inner components by sending them an optional classname instead of targeting their base class.